resources/views/shared/users/sidebar.blade.php

```diff
@@ -58,6 +58,22 @@
                 <div class="invalid-feedback" :style="{display: (errors.password) ? 'block' : 'none' }" role="alert"
                      v-for="(error, index) in errors.password">@{{error}}</div>
             </div>
+
+            @if (config('password-policies.2fa_enabled', false))
```

Comment:
If `password-policies.2fa_enabled` is enabled but no methods are enabled, this will show a label without options to enable.